src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule333orderingandspacing/InputCustomImportOrderValid.java

```diff
@@ -6,7 +6,6 @@
 import com.puppycrawl.tools.checkstyle.checks.design.FinalClassCheck;
 import com.puppycrawl.tools.checkstyle.checks.design.ThrowsCountCheck;
 import com.puppycrawl.tools.checkstyle.checks.design.VisibilityModifierCheck;
-import com.sun.accessibility.internal.resources.*;
```

Comment:
Since this is the import order check, can we find a replacement import to go in it's place?

Reply:
I doubt it. Take into account the issue #4845

Reply:
@pbludov This removed line doesn't have anything to do with #4845 .
`com.sun.` is not checkstyle production code or other checkstyle inputs. That issue was about directly connecting inputs of 2 different checks together.
If we did create a `com.sun` package, each one would be named different for each input that needs it.

Reply:
At this time, it is better to close this issue. This could be a stopper for JDK10, but now this is causing more problems than solves.